File: test/Movie/new/test03.py
#!/usr/bin/python
from ming import *

m =  SWFMovie();


# SWF_SETBACKGROUNDCOLOR 
m.setBackground(0xff, 0xff, 0xff);

# SWF_SHOWFRAME 
m.nextFrame(); # end of frame 1 

# SWF_END 

m.save("test03.swf");

File: test/Movie/new/test03.c
#include <libming.h>

static char *test = "test03";

main()
{
SWFMovie m;

m= newSWFMovieWithVersion(5);
SWFMovie_setBackground(m, 0xff, 0xff, 0xff);
SWFMovie_save(m,"test03.swf");

return 0;
}
